internal/pkg/store/ssm/ssm.go

```diff
@@ -66,6 +69,9 @@ func NewStore() (*SSM, error) {
 	return &SSM{
 		systemManager: ssm.New(sess),
 		identity:      identity.New(sess),
+		// If the region is not set to us-east-1, we can't find out if a domain name exists in the account.
+		// See https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DNSLimitations.html#limits-service-quotas
+		domains:       route53domains.New(sess, aws.NewConfig().WithRegion("us-east-1")),
```

Comment:
It seems a little weird that there is what seems like a client for another service within the SSM client here, but not sure if there's another way...

Reply:
yeah I agree maybe @kohidave can chime in, but I think the original goal behind the `store` package is for it to be agnostic of which AWS service we use. 

Maybe we can refactor by getting rid of the `ssm` sub-package and moving these files one level above (in a follow-up PR).

What do you think ?

Reply:
I think `ssm` and `route53domains` should be isolated and wrapped similarly to some of our other AWS dependencies and the `store` package should orchestrate dependencies to do whatever it needs.

Reply:
I added an additional commit to remove the "ssm" package by moving this functionality into one level above (the "store" package)